core/templates/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.ts

```diff
@@ -441,11 +442,11 @@ angular.module('oppia').directive('audioTranslationBar', [
               $scope.contentId, $scope.languageCode);
             if (audioTranslationObject) {
               $scope.isAudioAvailable = true;
-              $scope.isLoadingAudio = true;
+              $scope.audioIsLoading = true;
               $scope.selectedRecording = false;
               $scope.audioNeedsUpdate = audioTranslationObject.needsUpdate;
               $scope.durationSecs =
-                Math.round(audioTranslationObject.duration_secs);
+                Math.round(audioTranslationObject.durationSecs);
```

Comment:
Is this change verified? I don't see a corresponding change to the object definition, so was this not working before?

Reply:
Yes I because at one time I wanted to make sure my previous PR did the job. I found out that the object definition was not matching from this end. So that's why I made this change. I'll double check this just in case.